### ISSUE TITLE
how to use rust-openssl cryptoprovider with rustls ?

### ISSUE THREAD

**Issue by sarath3192** [sfackler/rust-openssl#2192]
Is this possible?
If yes, what I need consider while doing this?
Thank you in advance.

**Comment by sfackler** [comment]
I don't understand the question. rust-openssl and rustls are separate TLS implementations that don't interact.

**Comment by sarath3192** [comment]
Sorry,I mean to say. is it possible to use TLS from rustls and crypto provider from rust-openssl binding?

**Comment by tofay** [comment]
Yes, it's possible. I made a rustls crypto provider that uses rust-openssl at https://github.com/tofay/rustls-openssl.

The main thing missing was TLS PRF API, for which I raised https://github.com/sfackler/rust-openssl/pull/2329
